src/typings.ts

```diff
@@ -106,4 +102,10 @@ export type ReminderProposal = ProposalExtended & {
   suggestion: (ProposalExtended & { corrections: Proposal[] }) | null;
 };
 
-export type Proposals = Correction | Suggestion | Report;
+export type ReportExtended = Report & {
+  suggestion: Suggestion;
+  approvals: Approval[];
+  disapprovals: Disapproval[];
+};
```

Comment:
```suggestion
interface GodfathersDecisions {
  approvals: Approval[];
  disapprovals: Disapproval[];
}

export interface ReportExtended extends Report, GodfathersDecisions {
  suggestion: Suggestion;
};
```
Pense également à utiliser le typage `GodfathersDecisions` dans `ProposalExtended`
```ts
export type ProposalExtended = Proposal & GodfathersDecisions;
```